strawberry/ext/mypy_plugin.py

```diff
@@ -277,7 +280,10 @@ def add_method_to_class(
             cls.defs.body.remove(sym.node)
 
     self_type = self_type or fill_typevars(info)
-    function_type = api.named_type("__builtins__.function")
+    if isinstance(api, SemanticAnalyzerPluginInterface):
+        function_type = api.named_type("builtins.function")
+    else:
+        function_type = api.named_generic_type("builtins.function", [])
```

Comment:
Can we add a little comment here indicating that this logical block is to support static methods? It might help a future developer.

Reply:
this is is actually the diff between mypy 0.920 to 0.930. Locally i was testing against 0.920 until i realised it would break on 0.930. so i had to copy paste from 0.930 instead. 
honestly i'm not really sure why we need this if statement here....